### ISSUE TITLE
Improve handling `aria-*` props across all components

### ISSUE THREAD

**Issue by amje** [gravity-ui/uikit#1929]
In almost every component we use the same set of two props for labeling: `aria-label`, `aria-labelledby` and duplicate code every time. We can now support more props and remove duplication using new [interface](https://github.com/gravity-ui/uikit/blob/b82262e6ac3ac656f0d87a6275a081f9990aeb6a/src/components/types.ts#L72) and [helper](https://github.com/gravity-ui/uikit/blob/5cdc6753d4461b8e531870bcd9f4724dc6782d80/src/components/utils/filterDOMProps.ts).

Look at [PinInput](https://github.com/gravity-ui/uikit/blob/b82262e6ac3ac656f0d87a6275a081f9990aeb6a/src/components/PinInput/PinInput.tsx#L272) as a reference

**Comment by blasdfaa** [comment]
can i take this?

**Comment by blasdfaa** [comment]
> @blasdfaa are you still interested in taking on this task? It's quite specific, so perhaps you'd prefer to choose something else?

i dont care :) can I take this then - https://github.com/gravity-ui/uikit/issues/1905

**Comment by lolmaus** [comment]
Moving a discussion from Telegram to this issue per @korvin89's instruction.

Context: I asked for help finding a universal approach for passing arbitrary HTML attributes to any component. I wanted such an approach for my local components but wanted to keep it in sync with Gravity's approach. The specific case was for testing purposes but the discussion grew to the general problem of passing attributes. @korvin89 told me that it was not appropriate to discus the matter in a Telegram group dedicated to testing matters and gave me a link to this issue.

***

@korvin89, this issue's reference implementation only supports *four* ARIA attributes:
https://github.com/gravity-ui/uikit/blob/v6.34.0/src/components/utils/filterDOMProps.ts#L6-L9

There are over 50 [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes) that might be arbitrarily required on root elements of any component.

So am I supposed to create a feature request for every individual attribute whenever I need it? And you say I must provide justification for usage of each individual attribute?

And you already denied my suggestion to add support for passing `tabindex` to any component because there exists a component that overrides `tabindex`.

***

Is this the official position of Gravity maintainers? If I have a case where I need to, say, add `aria-details` to a `<Text/>`, am I expected to start a separate issue and *make a case* justifying my need to use this attribute on this particular component? And I may be denied if @korvin89 or another maintainer decides that my case is not common enough (or whatever the acceptance criteria are)?

Same question for non-ARIA attributes such as `draggable`, `tabindex`, `title`, `lang`, `dir`, `hidden`, `contenteditable`, `spellcheck`, etc.

***

TL/DR: I wish I was able to pass any universal HTML attribute to the root element of any component, without soul-crushing bureaucracy that slows down feature development cycle from hours to weeks. Is it such a crazy thing to ask?

**Comment by korvin89** [comment]
@lolmaus this issue is not relevant to your feature request. I've shown them just for your information.
